pytripgui/field_editor/__init__.py

```diff
@@ -0,0 +1,7 @@
+from pytripgui.field_editor.field_view import FieldQtView
+from pytripgui.field_editor.field_cont import FieldController
+
+# from https://docs.python.org/3/tutorial/modules.html
+# if a package's __init__.py code defines a list named __all__,
+# it is taken to be the list of module names that should be imported when from package import * is encountered.
+__all__ = ['FieldQtView', 'FieldController']
```

Comment:
Do we need such behaviour ? In my opinion inside GUI we would rarely do `from XX import *`, hence I'd drop the line with `__all__`

Reply:
If the model is not updated, then I think the other button should better be named "Cancel" rather than "Close". Assuming both buttons will close the window anyway.

(Generally, I might have prefered to have the model immediatly updated, as this is more in line with "modern" UI behaviour. But this is of secondary importance now.)

Reply:
It is named Cancel now with current code:
![image](https://user-images.githubusercontent.com/7374733/60186252-b178a400-982b-11e9-869d-bac648fad582.png)

Reply:
it is because flake complains about imports that werent used